docs/cellprofiler.rst

```diff
@@ -2,49 +2,52 @@ Analyze OMERO data using a Jupyter Notebook
 ===========================================
 
 
-**Description**
----------------
+Description
+-----------
 
 We will demonstrate how to integrate CellProfiler and
 OMERO using the CellProfiler Python API and the OMERO Python API. We
 will use a Jupyter notebook to demonstrate the integration.
 
 We will show:
 
--  How to adjust an existing CellProfiler pipeline so that it can be used with OMERO.
+- How to adjust an existing CellProfiler pipeline so that it can be used with OMERO.
 
--  How load images from a Plate using the OMERO API.
+- How load images from a Plate using the OMERO API.
 
--  How to run CellProfiler using its Python API.
+- How to run CellProfiler using its Python API.
 
--  How to plot the results.
+- How to plot the results.
 
--  How to save the generated results back to OMERO as OMERO.table so they can be used later on by OMERO.parade.
+- How to save the generated results back to OMERO as OMERO.table so they can be used later on by OMERO.parade.
 
-**Resources**
--------------
+Resources
+---------
 
 We will use a CellProfiler example pipeline to analyse RNAi screening
 data from the Image Data Resource (IDR).
 
--  PercentPositive Pipeline \ https://cellprofiler.org/examples/
+- Example pipeline `PercentPositive Pipeline <https://cellprofiler.org/examples/>`_.
 
--  IDR data \ https://idr.openmicroscopy.org/webclient/?show=screen-102
+- Images from IDR `idr0002 <https://idr.openmicroscopy.org/webclient/?show=screen-102>`_.
 
--  Notebook :download:`idr0002_save.ipynb <../notebooks/idr0002_save.ipynb>`
+- Notebook :download:`idr0002_save.ipynb <../notebooks/idr0002_save.ipynb>`
 
 For convenience, the IDR data have been imported into the training
 OMERO.server. This is only because we cannot save results back to IDR
 which is a read-only OMERO.server.
 
-**Step-by-Step**
-----------------
+Step-by-Step
+------------
+
+The pipeline is run on all 2-Channel images from each Well in the 96-well plate (each Well contains one image), generating a CSV file containing rows for different objects identified in the
+image and columns for various parameters measured.
 
 #. First, open the webclient and find the Plate belonging to trainer-1 named plate1_1_013.
 
-#. Go to \ https://mybinder.org/
+#. Go to `mybinder.org <https://mybinder.org/>`_.
 
-#. Copy and paste the URL of this repository
+#. Copy and paste the URL of this repository.
```

Comment:
When I am reading the text of the guide, I will have no chance to realize where is the hosting repository and what is its link.

Reply:
(it makes sense only now as we are in github here, but otherwise, for example finiding https://omero-guides.readthedocs.io/en/latest/cellprofiler/docs/cellprofiler.html via internet search engine - I have no way to go back to this repo